src/dosbox.cpp

```diff
@@ -812,6 +812,19 @@ void DOSBOX_Init(void) {
 	Pbool = secprop->Add_bool("umb",Property::Changeable::WhenIdle,true);
 	Pbool->Set_help("Enable UMB support.");
 
+    pstring = secprop->Add_string("ver",when_idle,"");
+    pstring->Set_help("Set DOS version (5.0 by default). Specify as major.minor format.\n"
+			"A single number is treated as the major version.\n"
+			"Common settings are 3.3, 5.0, 6.22, and 7.1.\n"
```

Comment:
I /think/ the conf system has (or had?) provision to handle float values.  You could then bound the range to say, (1.0, 7.2), similar to int conf values -- and those legal limits are printed for you in the generated .conf file.

If you can't bound the range programmatically, then at least say it - and tell what happens if it's outside. `"Range is 1.0 to 7.2, otherwise defaulting to 6.22".

Reply:
@kcgen This is just the DOS version to report. I don't really think there is a precisely defined range for this. You can for example set it to 11.0 and no problem, which is already supported by the current VER SET command before this PR. The listed values are just some common ones, but they does not imply any ranges at all.

Reply:
Makes sense; pass this string as-is to VER_SET, and have it take care of assessing and handling the value (or erroring out).

Reply:
Done.